src/transformState.js

```diff
@@ -5,7 +5,34 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const changedState = state;
```

Comment:
There is no need to create this variable, use function param when the function is mutable

Reply:
deleted.